core-tests/shared/src/test/scala/zio/ZManagedSpec.scala

```diff
@@ -144,7 +143,7 @@ object ZManagedSpec extends ZIOBaseSpec {
         }
       } @@ TestAspect.nonFlaky
     ),
-    suite("fromEffect")(
+    suite("apply")(
```

Comment:
Stray change?

Reply:
You're right 😄 I was going to muck with these, but I _did_ roll them back.
```suggestion
    suite("fromEffect")(
```